tower/TempTower.scad

```diff
@@ -0,0 +1,110 @@
+// Temp Tower
```

Comment:
Calibration tests are not necessary to test a slicer. I consider this out of scope.